backend/src/main/scala/cromwell/backend/BackendJobExecutionActor.scala

```diff
@@ -54,5 +53,5 @@ trait BackendJobExecutionActor extends BackendJobLifecycleActor with ActorLoggin
   /**
     * Abort a running job.
     */
-  def abortJob: Future[JobAbortResponse]
+  def abortJob: Unit
```

Comment:
I was thinking it would be useful only if it fails. The engine could decide to try again, or just fail the call, or do nothing, but at least it would have the information.